__tests__/commands/install/integration.js

```diff
@@ -212,6 +212,26 @@ test.concurrent('install file: protocol with relative paths', (): Promise<void>
   });
 });
 
+test.concurrent('install file: protocol without cache', async (): Promise<void> => {
+  const fixturesLoc = path.join(__dirname, '..', '..', 'fixtures', 'install');
+  const compLoc = path.join(fixturesLoc, 'install-file-without-cache', 'comp', 'index.js');
+
+  return fs.writeFile(compLoc,'foo\n').then(() =>
```

Comment:
No reason to mix promises and async/await.
await fs.writeFile would work here, too